Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -418,16 +418,6 @@ protected void Reset()
         /// </summary>
         protected virtual void Awake()
         {
-#if UNITY_INPUT_SYSTEM_ENABLE_VR && ENABLE_VR
-            if (HasStereoCamera(out var cameraComponent))
```

Comment:
You should also be able to delete the `private` `HasStereoCamera` method too since it isn't called anywhere else other than these `Awake` and `OnDestroy` methods.